typescript/infra/config/environments/mainnet3/owners.ts

```diff
@@ -80,6 +80,9 @@ export const ethereumChainOwners: ChainMap<OwnableConfig> = Object.fromEntries(
           validatorAnnounce: DEPLOYER, // unused
           testRecipient: DEPLOYER,
           fallbackRoutingHook: DEPLOYER,
+          // Ensure we know keep the Safe and ICA addresses for each chain somewhere in the config
+          safeAddress: safes[local],
```

Comment:
nit: agree plumbing this through here is the easiest, and also agree it's a bit of a hack 😄

I don't feel too strongly but wondering if it's easy to change some part of this to understand that these two keys are different from the others?

Like it'd be nice to have ownerOverrides.contractOverrides.foo or something. Or maybe some naming schema to make it even more clear that safeAddress is special? like `_safeAddress` or something

Reply:
to resolve this + the multiple safe edge case, I think we could update the schema we use here to something like

ownerOverrides.safe.owner 
ownerOverrides.ica.owner

and then we could have variations for individual contracts e.g.

ownerOverrides.safe.proxyAdmin
ownerOverrides.ica.proxyAdmin
etc

imo there needs to be a bit more of a rework to support this multi-safe world so the above proposal could be a stopgap til we figure out how to do it better